### DISCUSSION TITLE
HmIP-MOD-HO: Lüftungsposition und Licht

### DISCUSSION THREAD

**Discussion by TOBx2** [SukramJ/aiohomematic#462] [Q&A] [answered]
Hallo zusammen,

seit knapp zwei Monaten habe ich die hahomematic-Integration im Einsatz und bin sehr zufrieden. Der Wechsel von der "integrierten" Integration auf hahomematic verlief auch sehr reibungslos.
Daher ein großes Dankeschön für die Entwicklung dieser tollen Integration!!
(Auffällig ist v.a. die deutlich schnellere Rückmeldung der Zustände der Türschlossantriebe (sowohl bei HmIP-DLD, als auch bei KeyMatic). Mit der "original" Integration dauerte es ca. 30-45 Sekunden, bevor ein veränderter Status in HA erschien. Dies erfolgt nun innerhalb weniger Sekunden.)

Leider gibt es noch zwei Kleinigkeiten, die hahomematic nicht abbilden kann (oder ich noch nicht gefunden habe, wie ich es "aktiviere"?!).

Das "Homematic IP Smart Home Modul für Hörmann-Antriebe" (HmIP-MOD-HO) erlaubt neben den (bereits integrierten) Funktionen
- Öffnen, 
- Schließen und
- Status

die folgenden Funktionen
- **Lüftungsposition** und
- **Ein-/Ausschalten des integrierten Lichts** des Garagentorantriebs
(bspw. Supramatic E 4 -> https://www.hoermann.de/private-bauherren-und-modernisierer/antriebe/garagentor-antriebe/).

Die **Lüftungsposition** wird sogar korrekt im Zustand DOOR_STATE als ventilation_position angezeigt; leider kann sie jedoch nicht per HA angesteuert werden (bzw. ich habe es nicht gefunden).

Die **Beleuchtung** (wird in der CCU3 als Kanal 2 angezeigt) wird weder angezeigt, noch kann sie geschaltet werden.

Ist es –für mich als Anwender– möglich diese Funktionen per Konfiguration zu ergänzen oder muss dies in hahomematic programmiert werden?
(falls zweitens, würde ich mich sehr über eine entsprechende Erweiterung freuen und werde selbstverständlich gerne mit Diagnoseinfos, Testen, etc. unterstützen.)

Danke vorab und VG
  Tobias

**Comment by SukramJ** [comment, marked as answer]
HA kennt kein Lüftungsposition. Der implementierte Workaround ist daher mit dem Slider oder einem Serviceaufruf eine  Position zwischen 10 und 50 zu wählen (<10 geschlossen, >50 offen).
Der Schalter für das Licht sollte auch vorhanden sein. Mach doch mal einen Screenshot vom Gerät in HA.

**Reply by TOBx2** [reply]
Danke für die schnelle Antwort!

Tatsächlich; der "mysteriöse" State-Schalter ist das Licht. :) Da hätte ich auch mal selbst drauf kommen können...#Fehler40
(Ich hatte immer vermutete, dass das eine Art "toggle" für öffnen/schließen ist und habe daher bisher noch nie drauf gedrückt. )
![Bildschirmfoto 2022-05-26 um 12 53 39](https://user-images.githubusercontent.com/871120/170474239-1bac827f-2fd9-4d31-a2a1-ee721716396f.png)

Die Lüftungsposition wird auch immer korrekt angezeigt:
![Bildschirmfoto 2022-05-26 um 12 54 34](https://user-images.githubusercontent.com/871120/170474554-8be477b6-a1d1-44d1-b7fe-67a91016b553.png)
![Bildschirmfoto 2022-05-26 um 12 57 02](https://user-images.githubusercontent.com/871120/170474761-f656f579-12dc-406a-ae72-d51e20c6374e.png)

Aber das ist dann vermutlich der hahomematic-Workaround, um zumindest den Status korrekt zu reporten?! Mit "HA kennt keine Lüftungsposition" meinst Du die Cover-Entität von HA, vermute ich?!

Danke für den Tipp mit dem Slider-/Service-Call-Workaround!

**Reply by oekinger** [reply]
> HA kennt kein Lüftungsposition. Der implementierte Workaround ist daher mit dem Slider oder einem Serviceaufruf eine Position zwischen 10 und 50 zu wählen (<10 geschlossen, >50 offen). <

Danke, Danke, Danke dafür. Interessanterweise wird die Position Lüftungspostion (gesetzt über CCU) in Hassio mit Position 10 interpretiert. Setzt man 10 aber manuell, öffnet sich das Tor kein Stück, erst ab 11 bewegt es sich). Ich hatte das deswegen über eine Variable in der CCU gelöst, die direkt die Lüftunsposition angefahren hat. Schade dass Homematic keine individuelle Öffnung unter 10% zulässt.

**Reply by SukramJ** [reply]
Korrekt ist geschlossen <= 10.
Die Werte sind eigentlich beliebig gewählt, da ein HmIP-MOD-HO nur 3 Zustände kennt (offen, Lüftung, geschlossen).
